api-java-mixin.raml

```diff
@@ -280,11 +285,20 @@ types:
   ProductDiscountUpdateAction:
     (java-extends): 'com.commercetools.api.models.ResourceUpdateAction<ProductDiscountUpdateAction>'
   ProductType:
-    (java-extends): 'ProductTypeMixin, com.commercetools.api.models.DomainResource<ProductType>, com.commercetools.api.models.Referencable<ProductType>, com.commercetools.api.models.ResourceIdentifiable<ProductType>, com.commercetools.api.models.WithKey'
+    (java-extends): 'AttributeDefinitionContainer, ProductTypeMixin, com.commercetools.api.models.DomainResource<ProductType>, com.commercetools.api.models.Referencable<ProductType>, com.commercetools.api.models.ResourceIdentifiable<ProductType>, com.commercetools.api.models.WithKey'
     (java-mixin): |
       public static com.commercetools.api.models.common.ReferenceTypeId referenceTypeId() {
           return com.commercetools.api.models.common.ReferenceTypeId.PRODUCT_TYPE;
       }
+
+      default com.commercetools.api.models.product_type.AttributeDefinition getAttribute(final String attributeName) {
+          return AttributeDefinitionContainer.super.getAttribute(attributeName);
+      }
+
+      @Override
+      default Optional<AttributeDefinition> findAttribute(final String attributeName) {
+          return AttributeDefinitionContainer.super.findAttribute(attributeName);
+      }
```

Comment:
Not needed as you are just overriding pass through to the AttributeDefinitionContainer

Reply:
now I got what you meant
Thanks :)